sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -612,7 +612,7 @@ object ShowTableExtended {
     AttributeReference("namespace", StringType, nullable = false)(),
     AttributeReference("tableName", StringType, nullable = false)(),
     AttributeReference("isTemporary", BooleanType, nullable = false)(),
-    AttributeReference("information", StringType, nullable = false)())
+    AttributeReference("information", MapType(StringType, StringType, false), nullable = false)())
```

Comment:
I feel it's better to output a struct type here, so that users can easily know what fields they can query.

Reply:
I think it's a general problem that most command outputs are not query-able. `ESCRIBE EXTENDED tbl` is also for display only.